### PR TITLE
Better example scripts: Description --no-tune and --local-mode CLI options (autoregressive_action_dist.py)

### PR DESCRIPTION
**Pull request by stefanbschneider**
## Why are these changes needed?

Better documentation of RLlib examples and extra CLI args `--no-tune` and `--local-mode`. 
Here for the autoregressive action dist example.

**Open question @sven1977:** 
I added an option `--no-autoreg` (default False), which runs the example without the autoregressive actions for comparison.
Since action a2 depends on a1, I expected the version with autoregressive actions to learn much faster/better, but it's the other way around! Without autoregressive actions, the agent learns MUCH faster and also ultimately better results.
Do you know why?

I guess, in the given `CorrelatedActionsEnv`, learning the optimal actions just means a1=a2=obs, which is simple to learn even without autoregressive actions.
 
## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(
   - [x] Tested locally

### CODE REVIEW

**Comment by stefanbschneider** [comment]
@sven1977 I'm a bit surprised that the autoregressive actions lead to worse results than independent actions in the example here. Do you have an idea why? Am I doing sth wrong?

File: rllib/examples/autoregressive_action_dist.py [hunk 1]
```diff
@@ -8,84 +9,177 @@
 To do this, you need both a custom model that implements the autoregressive
 pattern, and a custom action distribution class that leverages that model.
 This examples shows both.
+
```
Comment (sven1977) [inline rllib/examples/autoregressive_action_dist.py:12]: Awesome!

File: rllib/examples/env/correlated_actions_env.py [hunk 2]
```diff
@@ -4,9 +4,19 @@
 
 
 class CorrelatedActionsEnv(gym.Env):
-    """Simple env in which the policy has to emit a tuple of equal actions.
+    """
+    Simple env in which the policy has to emit a tuple of equal actions.
 
-    The best score would be ~200 reward."""
+    In each step, the agent observes a random number (0 or 1) and has to choose
```
Comment (sven1977) [inline rllib/examples/env/correlated_actions_env.py:10]: Thanks for clarifying this here.

**Review by sven1977 — APPROVED** [review]
Thanks for the PR @stefanbschneider ! :)

**Pull request merged**